float32 upper_arm_roll_joint_effort
float32 shoulder_pan_joint_effort
float32 shoulder_lift_joint_effort
float32 forearm_roll_joint_effort
float32 elbow_flex_joint_effort
float32 wrist_flex_joint_effort
float32 wrist_roll_joint_effort

